crates/integration-tests/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "integration-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha",
+    "test:browser": "web-test-runner",
+    "test:node": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha",
+    "test:integration:browser": "web-test-runner test//integration/browser/**/*.test.ts",
+    "test:integration:browser:watch": "web-test-runner test/integration/browser/**/*.test.ts --watch"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@esm-bundle/chai": "^4.3.4-fix.0",
+    "@web/dev-server-esbuild": "^0.3.6",
+    "@web/test-runner": "^0.15.3",
+    "@web/test-runner-playwright": "^0.10.0",
```

Comment:
Seems like this is not used anymore. Can you explain why for context and remove this dep?

Reply:
`"@web/test-runner-playwright": "^0.10.0"` only, but left for above reasons

Reply:
they're not two, one is runner and another is needed to run playwright. I'd like to keep PW as quick on/off option.

Reply:
On occasion you may want/need to use one or the other. Playwright brings it's own headless browsers which in some environments use useful. WebDriver relies on browsers you have already installed in your system, eg. local system.